nixos/modules/services/misc/ombi.nix

```diff
@@ -31,6 +31,12 @@ in {
         description = lib.mdDoc "Open ports in the firewall for the Ombi web interface.";
       };
 
+      baseUrl = mkOption {
+        type = types.str;
+        default = "/";
+        description = lib.mdDoc "The base URL for the Ombi web interface.";
```

Comment:
`lib.mdDoc` is now just an alias and can be safely removed everywhere.
see https://github.com/NixOS/nixpkgs/commit/d36f950d405369abca5144b14a201bc25c4a080a and https://github.com/NixOS/nixpkgs/pull/237557

Reply:
Do you want me to remove it from all the options in the module, or only for the one I added?

Reply:
I removed everywhere in [106710b](https://github.com/NixOS/nixpkgs/pull/284311/commits/106710b5e1d01d871a527ef17bdd17cdc485df96)